liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -163,10 +173,18 @@ void SourceReferenceFormatter::printSourceLocation(SourceReference const& _ref)
 
 void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg)
 {
-	// exception header line
-	errorColored() << _msg.severity;
-	if (m_withErrorIds && _msg.errorId.has_value())
-		errorColored() << " (" << _msg.errorId.value().error << ")";
+    if (_msg.errorId.has_value())
```

Comment:
This line is still indented with spaces.

Reply:
Fixed 👍 

Can't I use clang-format file present in the source tree ?

Reply:
Unfortunately we did not manage to get `clang-format` to follow our style and we gave up on trying to get its config right. So it's better not to use it unless you're also willing to do some manual corrections afterwards (which defeats the whole point).